datumaro/plugins/transforms.py

```diff
@@ -351,6 +351,43 @@ def __iter__(self):
             yield self.wrap_item(item, id=i + self._start)
 
 
+class AnnotationReindex(ItemTransform, CliPlugin):
```

Comment:
How about naming `ReindexAnnotations` for the consistency within `Transform`?

Reply:
https://github.com/openvinotoolkit/datumaro/pull/1008/commits/1e058875a1068f6ba198d2c2c45f6be3ad7b6895 fixes it.